OpenRA.Mods.Common/Widgets/EditorViewportControllerWidget.cs

```diff
@@ -11,14 +11,28 @@
 
 using System;
 using OpenRA.Graphics;
+using OpenRA.Mods.Common.Traits;
 using OpenRA.Widgets;
+using Color = OpenRA.Primitives.Color;
 
 namespace OpenRA.Mods.Common.Widgets
 {
 	public class EditorViewportControllerWidget : Widget
 	{
 		public IEditorBrush CurrentBrush { get; private set; }
 
+		[Desc("Main color of the selection grid.")]
+		public readonly Color MainColor = Color.White;
```

Comment:
The names of these first three could ideally be made more specific and descriptive.